collector/spoolman.go

```diff
@@ -0,0 +1,55 @@
+package collector
+
```

Comment:
Add a comment with a link to the relevant section of the moonraker docs
i.e. `// https://moonraker.readthedocs.io/en/latest/web_api/#spoolman-apis`